.github/workflows/pr-helmchart-test.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Helm chart test for components
```

Comment:
better change it to 'helm test' to save display length in the final action GUI

Reply:
Will do